src/en/juju-concepts.md

```diff
@@ -0,0 +1,98 @@
+Title: Juju concepts
+
+
+!!! Note: This page is a work in progress, as part of the work towards 
+documenting Juju 2.0.
+
+# Juju concepts
+
+## The Juju client
+
+The Juju package which you install and run from your local computer is sometimes
```

Comment:
Suggestion:

The Juju client is CLI software that is used to manage Juju, whether as an administrator or as a regular user. It is installed, via an APT package (juju2), onto your personal workstation.

Consider mentioning Juju GUI?